### PR TITLE
RTL support by flag

### PR DESCRIPTION
**Pull request by rustyhamsterr**
## Guidelines
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

### Types of changes
What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Preparing a pull request for review
Ensure your change is properly formatted by running:

```gradle
$ ./gradlew spotlessApply
```

Please correct any failures before requesting a review.

### CODE REVIEW

**Comment by skydoves** [comment]
@rustyhamsterr 
Hi, thank you for your contribution!
I will check and review it soon.

File: README.md [hunk 1]
```diff
@@ -396,6 +396,7 @@ class ProfileBalloonFactory : Balloon.Factory() {
 .setLayout(@LayoutRes layout: Int)
 .setPreferenceName(value: String)
 .setShowTime(value: Int)
+.setRtlLayout(value: Boolean)
```
Comment (skydoves) [inline README.md]: `setRtlLayout` seems to cause confusion with `setLayout` and even it is a boolean type. 
How about `isRtlSupport`?

File: balloon/src/main/java/com/skydoves/balloon/Balloon.kt [hunk 2]
```diff
@@ -240,6 +243,11 @@ class Balloon(
     }
   }
 
+  private fun prepareLayoutDirection() {
+    horizontalLayoutFactor = if (builder.isRtl) -1
+    else 1
```
Comment (skydoves) [inline balloon/src/main/java/com/skydoves/balloon/Balloon.kt]: The default value is already `1`.
How about this?
```kotlin
if (builder.isRtl) {
  horizontalLayoutFactor = -1
}
```

File: balloon/src/main/java/com/skydoves/balloon/Balloon.kt [hunk 3]
```diff
@@ -817,6 +827,9 @@ class Balloon(
     /** sets the show times. */
     fun setShowTime(value: Int): Builder = apply { this.showTimes = value }
 
+    /** sets flag for using rtl layout */
+    fun setRtlLayout(value: Boolean): Builder = apply { this.isRtl = value }
```
Comment (skydoves) [inline balloon/src/main/java/com/skydoves/balloon/Balloon.kt]: Same. `setLayout` functionality is getting a `LayoutResource` as a parameter.
Maybe some developers would be confused with `setLayout`.

File: README.md (hunk 1 above)
Comment (rustyhamsterr) [reply README.md]: legit point, fixed

File: balloon/src/main/java/com/skydoves/balloon/Balloon.kt (hunk 2 above)
Comment (rustyhamsterr) [reply balloon/src/main/java/com/skydoves/balloon/Balloon.kt]: done

**Pull request merged**